persistence/model/src/main/java/gov/va/vro/persistence/model/ClaimEntity.java

```diff
@@ -30,18 +32,25 @@ public class ClaimEntity extends BaseEntity {
 
   @ManyToOne private VeteranEntity veteran;
 
+  // Multiple collections need to be eagerly fetched without causing a cartesian join product with
```

Comment:
This hasn't shown up int he testing so i thought since we only grab one claim at a time we avoided it. turns out we do not always, and this should be fixed sooner than later.